src/site/_includes/macros/authorcard.njk

```diff
@@ -0,0 +1,31 @@
+{% macro authorcard(collectionItem) %}
+
+{% set url = '/authors/'+collectionItem.key %}
+{% set data = collectionItem.data %}
+{% set thumbnail = data.thumbnail or data.hero or null %}
+{% set alt = data.alt or '' %}
+
+<div class="w-card w-card-author" role="listitem">
+  <article class="w-card-base">
+    <div class="w-card-base__cover w-card-base__cover--with-image }}">
+      <a href="{{url}}" class="w-card-base__link" tabindex="-1" aria-hidden="true">
```

Comment:
Would you mind adding a comment here that say: "Because we are not adding descriptive alt text to this image, and because the link is duplicated when we link the author's name below, we chose to hide this link from screenreaders to make their navigation a bit more efficient."